scripts/gd_to_csv/pddl_license.txt

```diff
@@ -0,0 +1,182 @@
+North American Mutual Aid Network Open Data Set
```

Comment:
if you could convert this to markdown it would be great to include it in a docs/ folder in github and I can link it in the readme

Reply:
@meganrm Okay, I have a .md version of the same file ready, plus removed one section which I shouldn't have included in either version.

Can you give me guidance on where the .md version should go?  I have it sitting in this same folder, but this folder isn't meant to be public-facing.  Are you planning to grab the .md file and put it somewhere else?

Reply:
this page still seems to have funky quote marks

Reply:
Oh, I didn't update the PR.  Still getting used to how to do this...

Reply:
@meganrm okay, figured it out.  New commit is now posted with all the changes I described in these comments.